src/main.ts

```diff
@@ -9,141 +7,189 @@ import generateSeqD from "./generateSeqD"
 import generateSD from "./generateSD"
 import generateTS from "./codeGen"
 import validateSeqD from "./validateSeqD"
+const inquirer = require("inquirer");
+
+const questions = [
+    {
+        type: "list",
+        name: "generationType",
+        message: "What's your desired generation type?",
+        choices: ["Sequence Diagram -> System Description", "System Description -> Sequence Diagram"],
+        filter(val) {
+            if (val === "Sequence Diagram -> System Description") {
+                return 0;
```

Comment:
Please use more human readable values for generation type (this comment is valid for some parts below)

Reply:
Done ✅